source/aks/upgrading-aks-clusters.html.md.erb

```diff
@@ -86,7 +91,7 @@ kubectl get pdb -A
 - To list all failed helm releases
 
 ```command
-kubectl get hr --all-namespaces | grep False
+kubectl get hr --all-namespaces | awk '/(Unknown|False)/'
```

Comment:
Do we change this to != True?

Reply:
make sense

Reply:
Something like that?
`kubectl get hr | awk '$3 != "True"' `

Reply:
think should be like this
`kubectl get hr --all-namespaces | awk '!/(True)/'`